inference.Rmd

```diff
@@ -39,7 +39,7 @@ By the end of the chapter, readers will be able to do the following:
 
 * Describe real-world examples of questions that can be answered with statistical inference.
 * Define common population parameters (e.g., mean, proportion, standard deviation) that are often estimated using sampled data, and estimate these from a sample.
-* Define the following statistical sampling terms (population, sample, population parameter, point estimate, sampling distribution).
+* Define the following statistical sampling terms: population, sample, population parameter, point estimate, and sampling distribution.
 * Explain the difference between a population parameter and a sample point estimate.
 * Use R to draw random samples from a finite population.
 * Use R to create a sampling distribution from a finite population.
```

Comment:
The question comes after Figure 10.1, not a big deal but looks a little odd

Reply:
fixed